pkg/server/register.go

```diff
@@ -200,8 +196,9 @@ func (i *InventoryServer) writeMachineInventoryCloudConfig(writer io.Writer, inv
 				SecretName:      inventory.Name,
 				SecretNamespace: inventory.Namespace,
 			},
-			Install: install,
+			Install: registration.Spec.Config.Elemental.Install,
 		},
+		Data: registration.Spec.Config.Data,
```

Comment:
At this point it is unclear to me if the `registration.Spec.Config.Elemental.Registration` has some valid or valuable data at this point, I guess there should not be anything in there and if so it can be simply ignored right?